### PR TITLE
Updated README and setup_guide to reflect new branding

### PR DESCRIPTION
**Pull request by keikreutler**

### CODE REVIEW

**Comment by keikreutler** [comment]
recheckcla

**Comment by auryn-macmillan** [comment]
@keikreutler you have to post "I have read the CLA Document and I hereby sign the CLA" as a comment here, and then we can recheck the CLA.

File: README.md [hunk 1]
```diff
@@ -1,65 +1,72 @@
...
+
+This module allows for execution of transactions that have been approved through a Reality.eth question for execution. The question asked on Reality.eth consists of a proposal ID (e.g. an IPFS hash), which can be used to provide more information for the transaction to be executed, as well as an array of EIP-712-based transaction hashes that represent the transactions that should be executed.
+
+These two components (`proposalId` and `txHashes`) uniquely identify a question on the module. While it is possible to ask the same question with different Reality.eth question parameters, it is only possible to execute transactions related to a specific question once.
+
+Once the question on Reality.eth has resolved to "yes", meaning that the transactions should be executed, they are submitted to the immutable executor defined in the module. Transactions defined in questions that resolve to "no" or "invalid" cannot be executed by the module.
```
Comment (auryn-macmillan) [inline README.md:17]: `executor` is now called `avatar` and is no longer immutable.

File: README.md [hunk 2]
```diff
@@ -1,65 +1,72 @@
...
+
+This module is intended to be used with [Gnosis Safe](https://github.com/gnosis/safe-contracts), but it ultimately framework agnostic.
```
Comment (auryn-macmillan) [inline README.md:19]: EDIT
> but **is** ultimately framework agnostic.

File: README.md [hunk 3]
```diff
@@ -1,65 +1,72 @@
...
 
 ### Features
-- Submit proposals uniquely identified by a `proposalId` and an array of `txHashes`, to create a Realitio question that validates the execution of the connected transactions.
-- Proposals can be marked invalid by the `executor` using `markProposalInvalid` preventing the execution of the transactions related to that proposal
-- The Realitio question parameters (`templateId`, `timeout`, `arbitrator`) are set on the module by the executor
-- A `minimum bond` can be set that is required to be staked on a Realitio answer before the transactions can be executed
-- A `cooldown` can be specified representing the minimum time that needs to pass after the Realitio question has been answered before the transactions can be executed
+- Submit proposals uniquely identified by a `proposalId` and an array of `txHashes`, to create a Reality.eth question that validates the execution of the connected transactions.
+- Proposals can be marked invalid by the `executor` using `markProposalInvalid`, thereby preventing the execution of the transactions related to that proposal.
```
Comment (auryn-macmillan) [inline README.md:23]: EDIT: 
> Proposals can be marked invalid by the `owner`

File: docs/setup_guide.md [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
+The guide will use the Rinkeby ETH Reality.eth contract at [`0x3D00D77ee771405628a4bA4913175EcC095538da`](https://rinkeby.etherscan.io/address/0x3D00D77ee771405628a4bA4913175EcC095538da#code). Other network addresses can be found in the Truffle build folder on the [Reality.eth GitHub repo](https://github.com/realitio/realitio-contracts). For example, on Mainnet the ETH Reality.eth contract can be found at [`0x325a2e0f3cca2ddbaebb4dfc38df8d19ca165b47`](https://etherscan.io/address/0x325a2e0f3cca2ddbaebb4dfc38df8d19ca165b47#code).
 
-DISCLAIMER: Check the deployed Realitio contracts before using them.
+DISCLAIMER: Check the deployed Reality.eth contracts before using them.
 
 ## Setting up the module
 
-The first step is to deploy the module. Every DAO will have their own module. The module is linked to a DAO (called executor in the contract) and an oracle (e.g. Realitio). These cannot be changed after deployment.
+The first step is to deploy the module. The module is linked to one DAO (called executor in the contract) and an oracle (e.g. Reality.eth). These cannot be changed after deployment.
```
Comment (auryn-macmillan) [inline docs/setup_guide.md:19]: EDIT
> The module is linked to one target address (usually an Avatar like the Gnosis Safe) and an oracle (e.g. Reality.eth).
The oracle cannot be changed after depoyment.

**Review by auryn-macmillan — COMMENTED** [review]
A few small edits.

**Pull request closed without merging**